### PR TITLE
integrate quotes for the api

### PR DESCRIPTION
**Pull request by King-Benx**
**Delivers**
- Consumes API to show quotes

### CODE REVIEW

File: src/components/ErrorComponent.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const ErrorComponent = ({ loading, error }) => (
+  <div className="quote">
+    {loading && <span>Loading quote</span>}
+    {error && (
+      <span className="error">
+        Something went wrong and we failed to get the quote
+      </span>
+    )}
+  </div>
+);
```
Comment (Efiamotu-1) [inline src/components/ErrorComponent.js:12]: - [ ] Your error state is not being handled properly and as such the error message which you have used here is not being displayed to the user rather the image below is what is being shown when i removed a single letter from the media api. Please kindly handle your errors correctly so it is displayed on the screen for the user :+1: 
er

File: src/components/ErrorComponent.js
Comment (Efiamotu-1) [inline src/components/ErrorComponent.js:1]: - [ ] Also i noticed that when the page loads it generates two quotes that are being displayed on the screen, this is happening as a result of the `<React.Strictmode></React.Strictmode>` which wraps your app component in your index.js file, if you remove it only one quote would be displayed at the start of the webpage. see this [link](https://legacy.reactjs.org/docs/strict-mode.html) to learn more about this

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @King-Benx , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @King-Benx , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Fetched Data form API ✔️
- Well Structured Code ✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**